tests/unit/h_periodic/h_beat_test.py

```diff
@@ -1,4 +1,41 @@
-# pylint: disable=unused-import
-# This is for some dodgy test coverage
+import pytest
 
-from h_periodic.h_beat import celery
+from h_periodic.h_beat import asbool
+
+
+@pytest.mark.parametrize(
+    "s,expected_output",
+    [
+        ("t", True),
+        ("true", True),
+        ("y", True),
+        ("yes", True),
+        ("on", True),
+        ("T", True),
+        ("TRUE", True),
+        ("Y", True),
+        ("YES", True),
+        ("ON", True),
+        ("1", True),
+        ("t ", True),
+        (" true", True),
+        (" y ", True),
+        ("   yes  ", True),
+        ("on  ", True),
+        (None, False),
+        ("", False),
+        ("f", False),
+        ("false", False),
+        ("n", False),
+        ("no", False),
+        ("off", False),
+        ("F", False),
+        ("FALSE", False),
+        ("N", False),
+        ("NO", False),
+        ("OFF", False),
+        ("0", False),
+    ],
+)
```

Comment:
You could probably make this a little slicker by having:

```python
@pytest.mark.parametrize('uppercase', (True, False))
...
if uppercase:
    string = string.upper()
```

If you cared to you could pull a similar trick with:

```python
@pytest.mark.parametrize("prefix", ("", "   "))
@pytest.mark.parametrize("suffix", ("", "   "))
...

Reply:
Done: https://github.com/hypothesis/h-periodic/blob/d7d48f80b72a1d84b399d45e743711e3b25cc48d/tests/unit/h_periodic/h_beat_test.py#L6-L36